docs/nodes/logging.md

```diff
@@ -0,0 +1,174 @@
+---
+order: 7
+---
+
+## Logging
+
+Structured logging adds detail to logs and allows the node operator to better identify what they are looking for. Tendermint supports log levels on a global and per module basis. This allows the node operator to see only the information they need and the developer to hone in on specific changes they are working on. 
+
+## Configuring Log Levels
+
+
+There are three log levels, `info`, `debug` and `error`. These can be configured either through the command line via  `tendermint start --log-level ""` or within the `config.toml` file.
+
+```toml
+# Output level for logging, including package level options
+log-level = "info"
+```
+
+The default setting is a global `info` level. If you would like to set specific modules level log levels it can be done in the following format. 
+
+```toml
+# Output level for logging, including package level options
+log-level = "info,mempool:error"
+```
+
+or
+
+```sh
+tendermint start --log-level "info,mempool:error"
+```
+
+Here we are setting the mempool log level to `error`, but still retain the global info level. 
+
+## List of modules
```

Comment:
This seems like a weird thing to have in the logging docs - can we just link to a canonical list of modules somewhere else?

Reply:
I was thinking this as well, but it would be weird if you had to go somewhere else to find the name of the module that only applies to the logging layer. 

We don't have a list of "modules", it's not a common term within tendermint.

Reply:
IMHO I think listing them here is fine as long as we keep the descriptions concise.